### DISCUSSION TITLE
CefSharp File Version and Product Version

### DISCUSSION THREAD

**Discussion by isgori3** [cefsharp/CefSharp#3917] [General]
I'm working currently on the version 95 from CefSharp, but I notice something in the branch is typed CefSharp-95.7.141, but when compile and run the files are with other version with 94.4.20, and in branch is there, so it's OK or is a different for a specific problem?

![Version error](https://user-images.githubusercontent.com/83226165/144320489-fec55fce-5ef8-423c-a79c-582a352b2a51.PNG)

**Comment by amaitland** [comment]
For the official releases the build script sets the version number based on the GIT tag.

You can run  the build script and pass in a version number (or edit the file) and the version numbers will be updated. 

https://github.com/cefsharp/CefSharp/blob/cefsharp/95/build.ps1#L8

Usually I'll check in the version number changes, occasionally I forget.
